### ISSUE TITLE
Bring static labels to back

### ISSUE THREAD

**Issue by Lukas-bw** [Leaflet/Leaflet.label#14]
Hello,

I am showing a lot of static labels on my map. My current problem is, that if I open a Leaflet popup, I cant read all information because the labels are on top of that.

Is there any way to bring that popup to front or the labels to back?

Regards

**Comment by tomchadwin** [comment]
I still have labels overlapping popups. Has this issue reappeared?

**Comment by muziejus** [comment]
I, too, am having this problem, and this is with adding the `pane: 'markerPane'` option.

**Comment by muziejus** [comment]
I solved the problem at least for myself.
```
this._pane=this._source instanceof L.Marker?t._panes.markerPane:t._panes.popupPane
```
This suggests that if the source is not on the markerPane (say it's on the overlayPane, like a polygon would be), then the label is placed on the popupPane.

So I just changed this line of code to:
```
this._pane=this._source instanceof L.Marker?t._panes.markerPane:t._panes.markerPane
```
It works for my specific use case, in that I only ever have circleMarkers. Despite the documentation, which calls for a `pane: 'markerPane'` option, I haven't been able to get that to work. It was persistently putting my labels in the popupPane, and I think this snippet of code shows why. 

Again, this is 0.2.2-dev.
